utils/log_test.go

```diff
@@ -19,7 +19,7 @@ var levelStrings = map[utils.LogLevel]string{
 
 func TestLogLevelString(t *testing.T) {
 	for level, str := range levelStrings {
-		t.Run("level "+str, func(t *testing.T) { //nolint:goconst
+		t.Run("level "+str, func(t *testing.T) {
```

Comment:
You need to add the trace level to 
var levelStrings = map[utils.LogLevel]string{
...
}	utils.TRACE: "trace",